movies/src/tests/tests.hydra

```diff
@@ -0,0 +1,34 @@
+PREFIX schema: <http://schema.org/>
+PREFIX hydra: <http://www.w3.org/ns/hydra/core#>
+PREFIX mov: <https://hydra-movies.herokuapp.com/doc#>
+
+With Class schema:EntryPoint {
+    Expect Property hydra:collection {
+        Expect Type hydra:Collection
+        Expect Type mov:Movies
+    }
+
+    Expect Link hydra:collection {
+        Expect Status 200
+    }
+}
+
+With Class mov:Movies {
+    Expect Operation <https://hydra-movies.herokuapp.com/doc#create-movie> {
+        Invoke {
+            Content-Type "text/turtle"
+
+            ```
+            @prefix schema: <http://schema.org/>
+            @prefix mov: <https://hydra-movies.herokuapp.com/doc#>
+
+            <> a mov:Movie ;
+               schema:name "Fake Movie"
+            ```
+        } => {
+            Expect Status 201
```

Comment:
Keyword `Expect` in the response block feels redundant as the `=>` notation somehow implies it

Reply:
Not sure I understand your point. You mean to have it like below?

```diff
-Expect Status 201
+Status 201
```

The `Expect` keyword mostly keeps the statements consistent. All assertions are `Expect`. Dropping it would add unnecessary variety to the language IMO.

Reply:
> The Expect keyword mostly keeps the statements consistent.

I thought so. It's just that `N3` like notation of the `=>` (implies) makes the word `Expect` redundant. My understanding of the notation is:

> for a request of a given shape I expect a response of the expected shape

Reply:
I see. The N3 analogy is coincidental. I was rather thinking about function lambdas.